app/config.go

```diff
@@ -14,20 +14,54 @@
 
 package app
 
+import (
+	toml "github.com/pelletier/go-toml/v2"
+	"os"
+)
+
+const DefaultConfFile = "conf.toml"
+const ConfFileEnvVar = "CONFIG_FILE"
+
 type Config struct {
-	GCPConfig *GCPConfig
+	AccountManager  AMConfig
```

Comment:
nit: should we use pointers here as well?

Reply:
Actually I prefer all of these members to be non-pointers since they are not optional (can't be nil). The GCPConfig field is a pointer because it would be nil if another implementation is ever used. We also don't transfer ownership of these member variables, they are only borrowed by the modules.